app/model/RuleMatch.scala

```diff
@@ -19,7 +19,7 @@ object RuleMatch {
       fromPos = lt.getFromPos,
       toPos = lt.getToPos,
       message = lt.getMessage,
-      shortMessage = Option(lt.getShortMessage).filterNot(_.isEmpty),
+      shortMessage = Some(lt.getMessage),
```

Comment:
shortMessage isn't present in RegexPatternRule, so we defer to message here. We could add it, but until we know more about our modelling requirements I'm not sure there's a reason to worry about it.